src/clonepi

```diff
@@ -1091,6 +1091,17 @@ runPostSyncHook()
 	fi
 }
 
+initDest()
+{
+sfdisk --delete /dev/$DEST_DISK
+echo -e "size=+63M, type=e\ntype=5\nsize=+32M,type=83\nsize=+256M,type=c\ntype=83"| sfdisk $DEST_DISK
+partprobe
+mkfs.vfat ${DEST_DISK}1
+mkfs -t ext4 -F ${DEST_DISK}5
```

Comment:
This assumes there is a partition 5 - won't be the case for a vanilla Raspbian install (2 partitions).